mne_bids/write.py

```diff
@@ -1185,7 +1185,7 @@ def write_raw_bids(raw, bids_basename, bids_root, events_data=None,
         print('Copying data files to %s' % op.splitext(bids_fname)[0])
 
     # File saving branching logic
-    if convert or (anonymize is not None):
+    if (convert or (anonymize is not None)) and ext != '.vhdr':
```

Comment:
can we avoid this maybe by reordering branching?

Reply:
yes, I tried simplifying the branching logic. Let's see if tests pass ...